doc/classes/RenderingServer.xml

```diff
@@ -861,6 +861,15 @@
 				A copy of the canvas item will be drawn with a local offset of the mirroring [Vector2].
 			</description>
 		</method>
+		<method name="canvas_set_item_repeat">
+			<return type="void" />
+			<param index="0" name="item" type="RID" />
+			<param index="1" name="repeat_size" type="Vector2" />
+			<param index="2" name="repeat_times" type="int" />
+			<description>
+				A copy of the canvas item will be drawn with a local offset of the [param repeat_size] by the number of times of the [param repeat_times]. As the [param repeat_times] increases, the copies will grow outwardly from the origin texture.
```

Comment:
Unconvinced about this description but that'd be nitpicky.
```suggestion
				A copy of the canvas item will be drawn with a local offset of the [param repeat_size] by the number of times of the [param repeat_times]. As the [param repeat_times] increases, the copies will spread away from the origin texture.
```

Reply:
I'm fine with nitpicks about this. I'm not sure a better more concise way to say, "when you increase this number, if on the x axis, it will add a copy to the right for odd numbers and to the left for even numbers. On the y axis, it will ad a copy to the bottom for odd numbers, and above for even numbers." It bothers me too!

Reply:
I think that on the RenderingServer this kind of explanation is much more "tolerable" because users reading may infer it by themselves (you usually don't mess with servers unless you know what you're doing).
On the other hand, the Parallax2D node itself needs to be clearer for the opposite reason.

Reply:
Gotcha. I'll go more in depth for the node.